spark-jobs/src/main/scala/za/co/absa/enceladus/conformance/interpreter/OptimizerTimeTracker.scala

```diff
@@ -26,7 +26,7 @@ class OptimizerTimeTracker(inputDf: DataFrame, isWorkaroundEnabled: Boolean)(imp
   private val log = LoggerFactory.getLogger(this.getClass)
 
   private val maxToleratedPlanGenerationPerRuleMs = 100L
-  private val initialElapsedTimeBaselineMs = 300L
+  private val initialElapsedTimeBaselineMs = 250L
```

Comment:
Just curious why the change? 😉

Reply:
It just while observing the behavior of more jobs I saw that 300  ms initial baseline is a little too high.